zio-cli/jvm/src/main/scala/zio/cli/ConfigFileArgsPlatformSpecific.scala

```diff
@@ -0,0 +1,48 @@
+package zio.cli
+
+import java.io.IOException
+import zio._
+import java.nio.file.{Files, Path, Paths}
+import scala.io.Source
+
+object ConfigFileArgsPlatformSpecific extends ConfigFilePlatformSpecific {
+  def findPathsOfCliConfigFiles(topLevelCommand: String): Task[List[String]] = {
+    val filename   = s".$topLevelCommand"
+    val cwd        = java.lang.System.getProperty("user.dir")
+    val homeDirOpt = java.lang.System.getProperty("user.home")
+
+    def parentPaths(path: String): List[String] = {
+      val parts = path.split(java.io.File.separatorChar).filterNot(_.isEmpty)
+      (0 to parts.length)
+        .map(i => s"${java.io.File.separatorChar}${parts.take(i).mkString(java.io.File.separator)}")
+        .toList
+    }
```

Comment:
Does this work for `Windows`/`Linux`/`Mac`?  
I have a sneaking suspicion it might not work correctly for `/home/kalin/dev/a/b/c/.cmd`  
Might be safer to implement via `Path#getParent`?